.github/ISSUE_TEMPLATE/019-bug-misc.yml

```
@@ -52,6 +52,15 @@ body:
         - Other (Please specify in the next section)
     validations:
       required: false
+  - type: textarea
+    id: command
+    attributes:
+      label: Command line
+      description: >
+        Please provide the exact command you used to run llama.cpp. For example: `llama-server -m ... -c ...`, `llama-cli -m ...`, etc.
+      render: shell
+    validations:
+      required: true
```

```suggestion
      required: false
```

There are cases where this does not apply.